lib/ssl/test/ssl_key_update_SUITE.erl

```diff
@@ -192,22 +190,33 @@ keylog_server_cb(Config) ->
     end,
     Role = receive
                {keylog, #{items := TConKeylog0}} ->
-                   traffic_secret(TConKeylog0)
+                   traffic_secret_0(TConKeylog0)
            end,
     OppsitRole = opposite_role(Role),
     receive
         {keylog, #{items := TConKeylog2}} ->
-            OppsitRole = traffic_secret(TConKeylog2)
+            OppsitRole = traffic_secret_0(TConKeylog2)
     end,
+    ok = traffic_secret_1_and_2([{client,1}, {client, 2}, {server,1}, {server, 2}]).
+
+traffic_secret_1_and_2([]) ->
+    ok;
+traffic_secret_1_and_2([_|_] = List) ->
     receive
-        {keylog, #{items := UpdateKeylog}} ->
-            ["SERVER_TRAFFIC_SECRET_1" ++ _| _] = UpdateKeylog
+        {keylog, #{items := ["SERVER_TRAFFIC_SECRET_1" ++ _| _]}} ->
```

Comment:
here and below, I don't understand why you have both in pattern

1. `_N` is present in patter
2. but also you match on anything possibly concatenated to that `_N` string

Why?

Reply:
Because there is more information in the item that is not static.

Reply:
aha, ok.